app/build.gradle

```diff
@@ -11,12 +11,12 @@ android {
         applicationId "com.tari.android.wallet"
         minSdkVersion 24
         targetSdkVersion 29
-        versionCode 2
-        versionName "0.0.2"
+        versionCode 1
```

Comment:
You're undoing a previous commit here

Reply:
Thanks, have set it to 3 and "0.0.3" respectively